scripts/release.sh

```diff
@@ -111,27 +111,22 @@ done
 
 [ -z "${VERSION-}" ] && error "--version is not set"
 
-[ -z "${TAURI_PRIVATE_KEY-}" ] && error "$TAURI_PRIVATE_KEY is not set"
-[ -z "${TAURI_KEY_PASSWORD-}" ] && error "$TAURI_KEY_PASSWORD is not set"
+[ -z "${TAURI_SIGNING_PRIVATE_KEY-}" ] && error "$TAURI_SIGNING_PRIVATE_KEY is not set"
+[ -z "${TAURI_SIGNING_PRIVATE_KEY_PASSWORD-}" ] && error "$TAURI_SIGNING_PRIVATE_KEY_PASSWORD is not set"
 
 if [ "$CHANNEL" != "release" ] && [ "$CHANNEL" != "nightly" ]; then
 	error "--channel must be either 'release' or 'nightly'"
 fi
 
-export TAURI_PRIVATE_KEY="$TAURI_PRIVATE_KEY"
-export TAURI_KEY_PASSWORD="$TAURI_KEY_PASSWORD"
-
 if [ "$DO_SIGN" = "true" ]; then
 	if [ "$OS" = "macos" ]; then
 		[ -z "${APPLE_CERTIFICATE-}" ] && error "$APPLE_CERTIFICATE is not set"
 		[ -z "${APPLE_CERTIFICATE_PASSWORD-}" ] && error "$APPLE_CERTIFICATE_PASSWORD is not set"
-		[ -z "${APPLE_SIGNING_IDENTITY-}" ] && error "$APPLE_SIGNING_IDENTITY is not set"
 		[ -z "${APPLE_ID-}" ] && error "$APPLE_ID is not set"
 		[ -z "${APPLE_TEAM_ID-}" ] && error "$APPLE_TEAM_ID is not set"
 		[ -z "${APPLE_PASSWORD-}" ] && error "$APPLE_PASSWORD is not set"
 		export APPLE_CERTIFICATE="$APPLE_CERTIFICATE"
 		export APPLE_CERTIFICATE_PASSWORD="$APPLE_CERTIFICATE_PASSWORD"
-		export APPLE_SIGNING_IDENTITY="$APPLE_SIGNING_IDENTITY"
```

Comment:
we dont need this anymore?

Reply:
I remember nightly failed to build unless I removed it, but I think we should investigate further.

https://github.com/tauri-apps/tauri/blob/a8105eccb2122f003b4a36bbfb747998f53c5f10/crates/tauri-cli/src/interface/rust.rs#L1299-L1306